packages/utils/src/line-items/prepare-line-item-data.ts

```diff
@@ -0,0 +1,56 @@
+import { ProductVariantDTO } from "@medusajs/types"
+import { MedusaError } from "medusa-core-utils"
+
+import pick from "lodash/pick"
+
+const VARIANT_PROPERTIES = [
+  "title",
+  "metadata",
+  "product_id",
+  "product.title",
+  "product.discountable",
+  "product.is_giftcard",
+]
+
+export type LineItemCreateData = {
+  variant_id: string
+  title: string
+  product_id: string
+  product: {
+    title: string
+    thumbnail: string | null
+    discountable: boolean
+    is_giftcard: boolean
+  }
+  metadata?: Record<string, unknown>
+
+  quantity: number
+  unit_price?: number
+}
+
+/**
+ * Extract from data need to generate line item from a variant.
```

Comment:
suggestion: Generate the expected shaped data to generate a line item from the variant data